.mailmap

```diff
@@ -10,3 +10,5 @@ Matthias Dellweg <2500@gmx.de> <mdellweg@redhat.com>
 Matthias Dellweg <2500@gmx.de> <dellweg@atix.de>
 Mark Hlawatschek <hlawatschek@atix.de> <mark@atix.de>
 Paul Gration <pmgration@gmail.com>
+Peter Ondrejka <pondrejk@redhat.com> <pondrejk@users.noreply.github.com>
+Richard <richielatk@gmail.com> <stempfl@atix.de>"
```

Comment:
```suggestion
Richard Stempfl <richielatk@gmail.com> <stempfl@atix.de>"
```

Reply:
Or isn't that how the mailmap works?

Reply:
that works, yeah. he never had his last name in the commits, that's why I ommited it originally.

Reply:
updated

Reply:
should we have his consent?

@IncredibleRichie

Reply:
It´s okay for me. :-)